app/assets/stylesheets/content/_select2.scss

```diff
@@ -126,3 +126,63 @@
     margin-left: 0;
   }
 }
+
+//@todo for select2 visibility in livingStyleGuide, need another solution  it will rewrite globally
+.livingstyleguide--example {
+  overflow: visible !important;
+}
+
+//$se2- select2 var prefix
+$se2-theme-selectable-color: #F8F8F8;
+//todo change to global theme color var
+$se2-theme-selectable-color-highlighted: #24B3E7; /*{bgThemeHighlight}*/
+
+$se2-element-height: 33px;
+$se2-line-height: 30px;
+$se2-button-width: 28px;
+$se2-arrow-button-width: 18px;
+$se2-results-max-height: 500px;
+
+.select2-container {
+  .select2-choice {
+    border-radius: 0;
+    height: $se2-element-height;
+    line-height: $se2-element-height;
+    box-shadow: none;
+    > span {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3